src/routes/safe/components/Apps/hooks/useAppsSearch.ts

```diff
@@ -0,0 +1,25 @@
+import { useMemo } from 'react'
+import Fuse from 'fuse.js'
+import { SafeApp } from 'src/routes/safe/components/Apps/types'
+
+const useAppsSearch = (apps: SafeApp[], searchText: string): SafeApp[] => {
+  const fuse = useMemo(
+    () =>
+      new Fuse(apps, {
+        keys: ['name', 'description'],
+        threshold: 0.2,
```

Comment:
What is threshold? Could you add a comment, maybe with a link to the Fuse.js docs?